tests/fixtures/inverter.cgi

```diff
@@ -0,0 +1 @@
+12345678910;3280031;205;38.4;780;23.799999;d;NO; 
```

Comment:
The fourth field (of value `38.4`) is not parsed anywhere. Any idea what it means, maybe inverter temperature (we have a field for that)?

Reply:
It is indeed the temperature. I somehow missed that parameter as part of the TCP values.
I'll populate it.